### PR TITLE
update to jdk10/11

### PR DESCRIPTION
**Pull request by notsureifkevin**
related to #102 

this will _probably_ work with v11, but I don't have a machine where I can test it out.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
         <docker.image.prefix>mszarlinski</docker.image.prefix>
         <docker.image.exposed.port>9090</docker.image.exposed.port>
         <docker.image.dockerfile.dir>${basedir}</docker.image.dockerfile.dir>
-        <docker.plugin.version>0.4.13</docker.plugin.version>
+        <docker.plugin.version>1.1.1</docker.plugin.version>
```
Comment (kcrawley) [inline pom.xml]: there was an attempt to fix the non-breaking docker build exception, didn't work, but I figured why not just keep it on the latest.

File: spring-petclinic-customers-service/pom.xml [hunk 2]
```diff
@@ -110,4 +125,28 @@
             </build>
         </profile>
     </profiles>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.hibernate.orm.tooling</groupId>
+                <artifactId>hibernate-enhance-maven-plugin</artifactId>
+                <version>5.2.17.Final</version>
```
Comment (kcrawley) [inline spring-petclinic-customers-service/pom.xml]: i'm open to suggestions on how to manage this build dependency - as it's required for hibernate to run in jdk9+ at runtime (tests will pass without it). i've tried a few things here:

* placing this stanza in parent pom didn't work
* referencing version info from parent pom didn't work
* leaving the version out generates a warning
* placing this inside the docker build plugin section means non-docker builds will generate jars which won't work during run time

**Comment by mszarlinski** [comment]
Thank you for the contribution, I will look into during weekend :)

**Comment by arey** [comment]
@notsureifkevin please see https://github.com/spring-petclinic/spring-petclinic-microservices/pull/130

**Pull request closed without merging**